### DISCUSSION TITLE
Dependencies for tracker_train.py

### DISCUSSION THREAD

**Discussion by larsmaurath** [DonsetPG/narya#24] [Q&A] [answered]
I have started to look into training my own player tracker with tracker_train.py but ran into some issues. Wanted to quickly check in if you also see some dependencies not imported (gluon, time, gcv, ...) that are also missing from requirements.txt or if I am doing something wrong.

Assuming that I get these dependencies installed, does this work for you as is?

Thanks a lot!

**Comment by DonsetPG** [comment, marked as answer]
Hey, 

Here is a link to a Colab I used to train the Gluon model: https://colab.research.google.com/drive/1HzakA4q5_s7NsNTKe5pbwSdQsXCRq8WB?usp=sharing

It doesn't use the narya library as I trained the model before writing the library. 
I'll try to set up a proper Colab with the training script from ```tracker_train.py``` when I have some time.

I hope the Colab allows you to train your model, I'll work on it otherwise.
Thanks!

**Reply by larsmaurath** [reply]
As always, thanks a lot Paul. This worked for me!
FYI, I did have to add the 'difficult' flag in the xmls, or else the creation of the datasets failed (when initializing 'VOCFootball').
